### PR TITLE
Added samples for Bedrock https call with Sigv4 in .NET

### PR DESCRIPTION
**Pull request by sssumarss**
*Issue #, if available:*

*Description of changes:*
- Added samples for Bedrock https call with Sigv4 in .NET
- Added support for http post request
- Refactored signing code into a separate class

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .DS_Store
Comment (jhart0) [inline .DS_Store:1]: Please remove these files. The .gitignore was mean to catch them.

File: no-sdk/.DS_Store
Comment (jhart0) [inline no-sdk/.DS_Store:1]: Please remove these files. The .gitignore was mean to catch them.

File: no-sdk/dotnet/.DS_Store
Comment (jhart0) [inline no-sdk/dotnet/.DS_Store:1]: Please remove these files. The .gitignore was mean to catch them.

File: no-sdk/dotnet/Program.cs
Comment (jhart0) [inline no-sdk/dotnet/Program.cs:1]: Can you add a conditional here so a user can choose which example they want to run i.e. likely one or the other

**Review by jhart0 — CHANGES_REQUESTED** [review]
Core changes look good - just a couple of minor comments

Can you also please update the readme so it's clear to viewers what each sample does / how to run each one.

**Comment by sssumarss** [comment]
Thanks for reviewing the code. Addressed all the review points above. Please have a look.

**Comment by jhart0** [comment]
LGTM!

**Pull request merged**